packages/components/src/styles/prose/prose.stories.ts

```diff
@@ -0,0 +1,709 @@
+            Facilisis commodo integer hendrerit tortor.
+          </p>
+        </div>
+        <!-- sd-prose/sample-h2-paragraph-h3-paragraph -->
+        <div class="sd-prose">
+          <h2>H2 Nisi eu excepteur anim esse</h2>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
+            phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
+            Facilisis commodo integer hendrerit tortor.
+          </p>
+          <h3>H3 Nisi eu excepteur anim esse</h3>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
+            phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
+            Facilisis commodo integer hendrerit tortor.
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
+            phasellusnon orci <a href="#">inline link goes here</a> pe dui vel id. Velit in sed non orci pellentesque
+            vivamus nunc. At non tortor, sit neque tristique. Facilisis commodo integer hendrerit tortor.
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
+            phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
+            Facilisis commodo integer hendrerit tortor.
+          </p>
+        </div>
+        <!-- sd-prose/sample-paragraph-group -->
+        <div class="sd-prose">
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
+            phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
+            Facilisis commodo integer hendrerit tortor.
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
+            phasellusnon orci <a href="#">inline link goes here</a> pe dui vel id. Velit in sed non orci pellentesque
+            vivamus nunc. At non tortor, sit neque tristique. Facilisis commodo integer hendrerit tortor.
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
+            phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
+            Facilisis commodo integer hendrerit tortor.
+          </p>
+        </div>
+        <!-- sd-prose/sample-image-caption-paragraph -->
+        <div class="sd-prose">
+          <figure>
+            <img src="./placeholders/generic.jpg" alt="A generic placeholder jpg" class="aspect-video object-cover" />
```

Comment:
Path has to be updated. cc @coraliefeil That's why the image is missing in Storybook.